sdk/src/framework/sc/shared/InvalidMoveException.kt

```diff
@@ -1,9 +1,12 @@
 package sc.shared
 
+import com.thoughtworks.xstream.annotations.XStreamAsAttribute
 import sc.api.plugins.IMove
 import kotlin.RuntimeException
 
 /** NOTE: It is important to have information about the move in the reason
  *  because that is the only place where the invalid move is logged */
```

Comment:
that note shouldn't be a doc comment, and you might as well remove it now

Reply:
3acc266c2a72dd26184c9689eedbe893a5d24dcc